src/frameworks/gemini/gemini-converter.ts

```
@@ -0,0 +1,155 @@
+  ): Promise<EvaluationRequest> {
+    let {
+      messages: requestMessages,
+      available_tools: requestAvailableTools,
+    } = await this.convertRequest(request);
+
+    const messages: LLMMessage[] = requestMessages || [];
+    const available_tools: LLMToolDefinition[] = requestAvailableTools || [];
+
+    // Check if response is streaming (has candidates array)
+    if (response?.candidates) {
+      let responseMessages = await this.handleResponse(response);
```

ESLint

[error] 'responseMessages' is never reassigned. Use 'const' instead.

(prefer-const)